src/test/scala/akka/persistence/query/journal/redis/EventsByPersistenceIdSpec.scala

```diff
@@ -67,7 +67,7 @@ class EventsByPersistenceIdSpec extends AkkaSpec(EventsByPersistenceIdSpec.confi
       src.map(_.event).runWith(TestSink.probe[Any])
         .request(2)
         .expectNext("a-1", "a-2")
-        .expectNoMsg(500.millis)
+        .expectNoMessage(500.millis)
```

Comment:
Grumpf, changing names in a patch version (I mean in akka, not here) is so bad...

Reply:
They didn't change completely, only deprecated the old name :) Still, I don't like to see warnings when I compile!

Reply:
Then, LGTM :)